spec/menu_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dish_name) { "Beef" }
+  let(:dish) { [{name: "Beef", price: 7, quantity: 3}] }
+  let(:order) { double(:order, add: dish) }
+  let(:order_klass) { double(:order_klass, new: order) }
+  subject(:menu) { described_class.new(order_klass) }
+
+  describe '#view' do
+    it 'should return a full list of dishes with prices' do
+      expect(menu.view).to start_with("---- MENU")
+    end
+
+    xit 'should return a list of appetisers with prices' do
+      expect(menu.view(appetisers)).not_to eq nil
+    end
+  end
+
+  describe '#select' do
+    it 'should return the selected dish with its price and quantity' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.